plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -63,62 +64,36 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
     console.clearConsole();
     console.activate();
 
-    LocalAppEngineServerDelegate serverDelegate = LocalAppEngineServerDelegate.getAppEngineServer(server);
-    int debugPort = -1;
-    if (ILaunchManager.DEBUG_MODE.equals(mode)) {
-      debugPort = getDebugPort();
+    setDefaultSourceLocator(launch, configuration);
```

Comment:
> Just out of curiosity: is this useful for a run launch as well? Not only for debug?

I'm not sure.  I imagine it might be used for stack trace hyperlinks?  I do wonder if we need to do something more for the Maven case where we should use m2e's `ISourceLookupDirector`.